doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -23,7 +28,120 @@
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| .il-ipsec | In-line IPSec acceleration | optional <sup>1)</sup> |
+| .la-crypto | Look-Aside encryption/decryption engine | optional <sup>2)</sup> |
+| .la-trans | Look-Aside Transcoding acceleration | optional <sup>3)</sup> |
+| .la-programmable | Look-Aside programmable acceleration | optional <sup>4)</sup> |
+
+**1)** In Kubernetes based infrastructures virtio-ipsec can be applied only if the CNI plugin is OVS.<br>
+**2)** In Kubernetes based infrastructures virtio-crypto can be applied only if the CNI plugin is OVS.<br>
+**3)** In Kubernetes based infrastructures virtio-trans can be applied only if the CNI plugin is OVS.<br>
+**4)** In Kubernetes based infrastructures virtio-programmable can be applied only if the CNI plugin is OVS.
+
+
+<a name="2.2.2"></a>
+### 2.2.2 Network Interface Options
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| n1, n2, n3, n4, n5, n6 | | Y |
+| n10, n20, n30, n40, n50, n60 | | Y |
+| n25, n50, n75, n100, n125, n150 | | Y |
+| n50, n100, n150, n200, n250, n300 | | Y |
+| n100, n200, n300, n400, n500, n600 | | Y |
+
+<a name="2.2.3"></a>
+### 2.2.3 VIM Capabilities
+| Attribute | Description | Value | Supported |
+|-----------|---------------------------|-------|-------|
+| e.vim.cap.001 | Virtual Compute allocation | | Y |
+| e.vim.cap.002 | Virtual Storage allocation | | Y |
+| e.vim.cap.003 | Virtual Networking resources allocation | | Y |
+| e.vim.cap.004 | Multi-tenant isolation | | N <sup>1)</sup> |
+| e.vim.cap.005 | Images management | | Y |
+| e.vim.cap.006 | Virtual resources inventory per tenant | | Y |
+| e.vim.cap.007 | Resources Monitoring | | Y |
+| e.vim.cap.008 | Virtual resources Performance  |  | Y |
+| e.vim.cap.009 | Virtual resources Fault information | | Y |
+
+**1)** In a Kubernetes based infrastructure within one Kubernetes cluster multitenacy is provided only in resource management level. Isolation of execution environments requires separate Kubernetes clusters or worker nodes.
+
+<a name="2.2.4"></a>
+### 2.2.4 Instance Capabilities Mapping
+
+| Attribute | Description | Value | Supported |
+|-----------|---------------------------|-------|-------|
+| e.nfvi.res.cap.001 | Max number of vCPU that can be assigned to a single VNFC | 16 | Y |
+| e.nfvi.res.cap.002 | Max memory in MB that can be assigned to a single VNFC by NFVI | 32 GB | Y |
+| e.nfvi.res.cap.003 | Max storage in GB that can be assigned to a single VNFC by NFVI | 320 GB | Y |
+| e.nfvi.res.cap.004 | # Connection Points | 6 | Y |
+| e.nfvi.res.cap.005 | Total instance (persistent) storage (GB | 300 GB | Y |
+| e.nfvi.per.cap.001 | CPU pinning support | | Y |
+| e.nfvi.per.cap.002 | NUMA support | | Y |
+| e.nfvi.per.cap.003 | IPSec Acceleration | | Yes (if offered |
+| e.nfvi.per.cap.004 | Crypto Acceleration | | Yes (if offered |
```

Comment:
e.nfvi.per.cap.004, e.nfvi.per.cap.005, e.nfvi.per.cap.006 and e.nfvi.per.cap.007 should refer to "2.2.1 Network Acceleration Extensions"